src/core/game.ts

```diff
@@ -119,6 +122,30 @@ export class Game {
     });
   }
 
+  static isFinished(game: GameState) {
+    const players = game.entities.players;
+    let playersAlive = Object.keys(players).length;
+
+    _.forEach(players, (player) => {
+      if (!player.alive) {
+        playersAlive--;
+      }
+    });
+    return (playersAlive === 1);
+  }
+
+  static getWinner(game: GameState) {
```

Comment:
```js
return _.find(game.entities.players, p => p.alive);
```

Don't have something like `game.isFinished = true` in a function that says `getWinner`.  You want to always ensure that what a function does matches what its name implies.